pages/hub/apps/sdk/data-hooks/useWaveStats.mdx

```diff
@@ -0,0 +1,60 @@
+import { Callout } from 'components'
+
+Hook is used to get the [Wave User Stats](/hub/apps/guides/azuro-waves#get-user-stats) data.
+
+<Callout type="info">
+  This hook is wrapper around [getWaveLevels](/hub/apps/sdk/utils/getWaveLevels) function.
```

Comment:
and here =)

Reply:
0f86ef07421d7347f86c7aca7a532616ae866ca9